pastes/views.py

```diff
@@ -1,3 +1,45 @@
+import json
 from django.shortcuts import render
+from django.http import HttpResponse
+from django.shortcuts import redirect
 
-# Create your views here.
+from .models import Pastes
+from . import tokens
+
+def list(request):
+    p = Pastes.objects.filter(user=request.user.id)
+    return render(request, 'pastes.html', {'pastes': p})
+
+def delete(request, id):
+    p = Pastes.objects.filter(id=id).delete()
+    return redirect('/')
+    
+
+def show(request, token):
+    if token.isdigit():
+        p = Pastes.objects.filter(id=token).values('paste')
+        if p is not None:
+            return render(request, 'show.html', {'paste': p[0]})
+    else:
+        p = Pastes.objects.filter(token=token).values('paste')
+        
+        if p is not None:
+            return render(request, 'show.html', {'paste': p[0]})
+
+def paste(request):
```

Comment:
Again, a view that has a clear path to not returning a value.

You might want to use :
https://docs.djangoproject.com/en/2.2/topics/http/decorators/#django.views.decorators.http.require_POST

Then you can avoid the `if` also.